Refreshing/going back while on demo crashes the server

Pretty annoying when I make a change and refresh. This is on Chrome v51:

```
❯ npm start

> xterm@0.33.0 start /home/daniel/dev/Tyriar/xterm.js
> bash bin/server

App listening to http://0.0.0.0:3000
close
/home/daniel/dev/Tyriar/xterm.js/node_modules/ws/lib/WebSocket.js:219
    else throw new Error('not opened');
         ^

Error: not opened
    at WebSocket.send (/home/daniel/dev/Tyriar/xterm.js/node_modules/ws/lib/WebSocket.js:219:16)
    at Socket.<anonymous> (/home/daniel/dev/Tyriar/xterm.js/demo/app.js:34:8)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at Pipe.onread (net.js:531:20)

npm ERR! Linux 4.4.0-22-generic
npm ERR! argv "/usr/local/bin/node" "/home/daniel/.npm-packages/bin/npm" "start"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! xterm@0.33.0 start: `bash bin/server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the xterm@0.33.0 start script 'bash bin/server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the xterm package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     bash bin/server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs xterm
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls xterm
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/daniel/dev/Tyriar/xterm.js/npm-debug.log
```

--- a/demo/app.js
+++ b/demo/app.js
@@ -29,9 +29,12 @@ app.ws('/bash', function(ws, req) {
     cwd: process.env.PWD,
     env: process.env
   });
-
   term.on('data', function(data) {
-    ws.send(data);
+    try {
+      ws.send(data);
+    } catch (ex) {
+      // The WebSocket is not open, ignore
+    }
   });
   ws.on('message', function(msg) {
     term.write(msg);

